### DISCUSSION TITLE
Upgrade to StartSharp Project

### DISCUSSION THREAD

**Discussion by volkankorkmaz** [serenity-is/Serenity#6808] [General]
Hi everyone,

We want to upgrade our project that we started with the Community project by paying for the StartSharp project. 

Can we directly update the existing project with DLL updates after payment?  Do you have any documantation for this?

Thank you,
Volkan

**Comment by VictorTomaili** [comment]
Yes it is possible
https://serenity.is/docs/startsharp/serene-upgrade-migrate
